demos/bower_components/fast-json-patch/.bower.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
If you need an example: https://github.com/felixfbecker/node-sql-template-strings/blob/master/.travis.yml#L25